docs/scos/dev/guidelines/operatable-feature-guidelines.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Operatable feature guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: NFR guidelines
+  link: docs/scos/dev/guidelines/nfr-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+---
+
+In order to enable Operations Teams to best-possible operate Applications, the following elements are suggested to be addressed.
+
+### Survive-ability
+An application - during its life-cycle - faces various expected and unexpected challenges. To increase its survive-ability with the help of Operations Team, 
+it is recommended to design additional flexible configuration options (feature or workflow disabling/switching) that can be adjusted on-the-fly 
+(without re-deployment) to decrease resolution impact on critical business flows (such options need to be handed over in the Operational guidelines).
+
+### Stability
+It is very common use-case that your business/technical teams identifies project specific risks that temporary challenge the stability of your application. 
+To enable us counter such events with a joined team effort or apply alternative solutions for these specific cases, it is recommended to hand over such 
+gathered information to us via the Operational guidelines.
```

Comment:
Also, to note the SES initiative here: defining and handing over Operational guidelines - is a very interesting part of the project lifecycle. It would be great to know more here from Aldi experince and think about offering that to our SES as a service OR as part of existing service